### PR TITLE
[rush] Add support for string parameter in custom commands

### PR DESCRIPTION
**Pull request by dotnetjunky**
This is to address issue #1189 : Allows custom commands to specify parameters of string types, in addition to `flag` and `choice`.

### CODE REVIEW

**Comment by dotnetjunky** [comment]
@acoates-ms Check this out.

File: apps/rush-lib/src/schemas/command-line.schema.json [hunk 1]
```diff
@@ -137,7 +137,7 @@
           "title": "Parameter Kind",
           "description": "Indicates the kind of syntax for this command-line parameter: \"flag\" or \"choice\"",
```
Comment (octogonz) [inline apps/rush-lib/src/schemas/command-line.schema.json]: Update the description

**Comment by chrisdothtml** [comment]
Has this change been released? I don't see it in the [documentation](https://rushjs.io/pages/configs/command_line_json/) or [schema](https://developer.microsoft.com/json-schemas/rush/v5/command-line.schema.json)

**Comment by chrisdothtml** [comment]
@dotnetjunky @octogonz anything? Would like to use this feature but don't know how. Thanks!

**Comment by octogonz** [comment]
@chrisdothtml  Sorry I missed your comment.  This feature was published with Rush 5.6.4; in order to use it, you would need to update `rushVersion` in your **rush.json** file.

I see it in the current [JSON schema](https://developer.microsoft.com/json-schemas/rush/v5/command-line.schema.json).

It does appear to be missing from the docs.  If you want to open a [GitHub issue](https://github.com/Microsoft/rushjs.io-website) for the web site, we can get the docs updated.

**Pull request merged**